ci/pipeline.yml

```diff
@@ -0,0 +1,191 @@
+#@ load("@ytt:data", "data")
+
+#@ def pipeline_image():
+#@   return data.values.docker_registry + "/galoy-pay-pipeline"
```

Comment:
Are there requirements here different than for the main repo? I'm thinking it might make sense to consolidate the images for all the node js pipelines by reusing the Galoy-main one. Like we are re-using the Galoy-deployments one for charts + Galoy-infra.

Reply:
Yep, I think we can re-use the galoy main one here. Fixed.